src/test/java/by/a1qa/tests/AddTicketToBasketTest.java

```diff
@@ -0,0 +1,65 @@
+package by.a1qa.tests;
+
+import aquality.appium.mobile.application.AqualityServices;
+import by.a1qa.data.Constants;
+import by.a1qa.models.Show;
+import by.a1qa.models.Ticket;
+import by.a1qa.screens.*;
+import by.a1qa.steps.TestSteps;
+import by.a1qa.testreport.ScreenshotListener;
+import by.a1qa.utils.ListUtils;
+import by.a1qa.utils.TicketUtils;
+import org.testng.Assert;
+import org.testng.annotations.Listeners;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+@Listeners(ScreenshotListener.class)
+public class AddTicketToBasketTest extends BaseTest {
+    private static ShowsScreen showsScreen = new ShowsScreen();
+    private static DiscoverScreen discoverScreen = new DiscoverScreen();
+    private static ShowPage showPage = new ShowPage();
+    private static TicketScreen ticketScreen = new TicketScreen();
+    private static LogInScreen logInScreen = new LogInScreen();
+
+    @Test
+    public void addTicketsToBasketTest() {
+        TestSteps.onboarding();
+
+        AqualityServices.getLogger().info("Step 1");
+        showsScreen.clickDiscoverButton();
+        Assert.assertTrue(discoverScreen.isOnDiscoverScreen(), "Isn't discover screen");
+
+        AqualityServices.getLogger().info("Step 2");
```

Comment:
add your own logger, so that you don't need to write `AqualityServices.getLogger().info("blah");`.
it should look like
```java
Logger.info("blah");
Logger.step(33, "Close calculator");